packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -217,6 +217,7 @@ export default class TradeStore extends BaseStore {
     non_available_contract_types_list: TContractTypesList = {};
     trade_type_tab = '';
     trade_types: { [key: string]: string } = {};
+    contract_types_list_v2: TContractTypesList = {};
```

Comment:
i believe this needs to be observable

Reply:
Yes, good catch, I forgot to make it an observable

Reply:
Added